computer_vision/object_detection/yolo_v8/run.py

```diff
@@ -0,0 +1,128 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2022, Ampere Computing LLC
+
+import argparse
+
+import torch
+from ultralytics.yolo.utils import ops
+
+from utils.cv.coco import COCODataset
+from utils.benchmark import run_model
+from utils.misc import print_goodbye_message_and_die
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="Run YOLOv8 model.")
+    parser.add_argument("-m", "--model_path",
+                        type=str, required=True,
+                        help="path to the model")
+    parser.add_argument("-p", "--precision",
+                        type=str, choices=["fp32"], required=True,
```

Comment:
precision choice is required although no choice available, yet framework choice is not required but is needed

Reply:
🤯